modules/claims_api/app/controllers/claims_api/v0/forms/intent_to_file_controller.rb

```diff
@@ -9,20 +9,26 @@ class IntentToFileController < ClaimsApi::BaseFormController
         skip_before_action(:authenticate)
         before_action :validate_json_format, if: -> { request.post? }
         before_action :validate_json_schema, only: %i[submit_form_0966 validate]
-        before_action :check_future_type, only: [:submit_form_0966]
 
         FORM_NUMBER = '0966'
 
         def submit_form_0966
-          response = itf_service.create_intent_to_file(form_type)
-          render json: response['intent_to_file'],
+          bgs_response = bgs_service.intent_to_file.insert_intent_to_file(intent_to_file_options)
+          render json: bgs_response,
                  serializer: ClaimsApi::IntentToFileSerializer
```

Comment:
will bgs_response ever be nil? we'll probably get a 500 if we pass nil to the IntentToFileSerializer

Reply:
It won't be nil during the actual submit, meaning we'll get something back even if it's an error from their end, but we should never pass nil if validate_json_schema and validate_json_format does it's job.